br/pkg/registry/registration.go

```diff
@@ -214,10 +218,19 @@ func NewRestoreRegistry(g glue.Glue, dom *domain.Domain) (*Registry, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	tableExists := true
+	_, err = dom.InfoSchema().TableByName(ctx, ast.NewCIStr(RestoreRegistryDBName), ast.NewCIStr(RestoreRegistryTableName))
+	if err != nil {
+		if !infoschema.ErrTableNotExists.Equal(err) {
```

Comment:
should we use err.Is() which is is standard errors package, it can auto unwrap if this error is wrapped with other error

Reply:
`infoschema.ErrTableNotExists.Equal` is widely used to handle the error from the interface `TableByName` in TiDB repository. Maybe it is enough to only use `Equal`.